node-tests/blueprints/ember-bootstrap-test.js

```diff
@@ -28,74 +28,151 @@ function createStyleFixture(name) {
 describe('Acceptance: ember generate ember-bootstrap', function() {
   setupTestHooks(this);
 
-  it('skips blueprint when no preprocessor present', function() {
-    let args = ['ember-bootstrap'];
-
-    return emberNew()
-      .then(() => emberGenerate(args, (file) => {
-        expect(file('app/styles/app.scss')).to.not.exist;
-        expect(file('app/styles/app.less')).to.not.exist;
-      }));
-  });
+  describe('import styles', function() {
+
+    describe('based on existing preprocessor', function() {
+
+      it('skips blueprint when no preprocessor present', function() {
```

Comment:
It doesn't skip the blueprint. It just won't create an `app.{less|scss}`. There will still be other effects, like an impact on the package to import and the build configuration.